events/2021-kubevirt-summit/proposals/prow-jobs-automated-testing.md

```diff
@@ -0,0 +1,38 @@
+# Title
+
+How to avoid merging broken code with automated testing using prow
+
+# Abstract
+
+This session is about creation of prow jobs in general. [Prow](https://github.com/kubernetes/test-infra/blob/master/prow/README.md) is a Kubernetes based CI/CD system.
+
+We will cover
+- what is a prow job
+- what job types are available
+- how to configure job triggers
+- testing jobs: requirements, obstacles and pitfalls
+- configurations for merge gating
+- other usages of jobs (publishing, bumping, ...)
+
+Attendants will be able to know what job types are available, which one to use for a specific problem and how to verify their job works on their machine (TM) before creating a PR.
+
+# Presenters
+
+- Daniel Hiller, Senior Software Engineer OpenShift Virtualization, Red Hat, [@dhill3r](https://twitter.com/dhill3r), [@dhiller (GitHub)](https://github.com/dhiller)
+
+[ ] The presenters agree to abide by the
+    [Linux Foundation's Code of Conduct for Events](https://events.linuxfoundation.org/about/code-of-conduct/)
+
+# Session details
+
+- Track: User | Contributors
```

Comment:
I assume this is for the Contributors track, right?

Reply:
Thanks for the catch! Fixed